api/config/config_perf.json

```diff
@@ -0,0 +1,73 @@
+{
+  "firecloud": {
+    "baseUrl": "https:\/\/firecloud-orchestration.dsde-perf.broadinstitute.org",
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-perf-",
+    "clusterMaxAgeDays": 7,
+    "clusterIdleMaxAgeDays": 3,
+    "debugEndpoints": false,
+    "enforceRegistered": true,
+    "leoBaseUrl": "https:\/\/leonardo.dsde-perf.broadinstitute.org",
+    "registeredDomainName": "all-of-us-registered-perf",
+    "registeredDomainGroup": "all-of-us-registered-perf@perf.test.firecloud.org",
+    "billingRetryCount": 4,
+    "billingProjectBufferCapacity": 20,
+    "xAppIdValue": "perf-AoU-RW",
+    "vpcServicePerimeterName": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_perf"
+  },
+  "auth": {
+    "serviceAccountApiUsers": [
+      "all-of-us-rw-perf@appspot.gserviceaccount.com"
+    ]
+  },
+  "cdr": {
+    "debugQueries": true
+  },
+  "googleCloudStorageService": {
+    "clusterResourcesBucketName": "all-of-us-rw-perf-cluster-resources",
+    "credentialsBucketName": "all-of-us-rw-perf-credentials",
+    "emailImagesBucketName": "all-of-us-workbench-test-email-images"
```

Comment:
Existing issue: why is this always coming from test? Should we add this to the deploy process?

Reply:
Not sure. I'm going to say out of scope for this though.

Reply:
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-3317&epics=visible&selectedEpic=RW-1801